packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -127,7 +127,7 @@ export const SuiteLayout = ({ children }: SuiteLayoutProps) => {
             <PageWrapper>
                 <ModalContextProvider>
                     <Metadata title={title} />
-                    <SuiteBanners />
+                    <SuiteBanner />
```

Comment:
technically, it indeed does rended multiple bannerS :)

Reply:
Done!